docs/site/Model.md

```diff
@@ -215,6 +215,11 @@ The complete list of valid attributes for property definitions can be found in
 LoopBack 3's
 [Model definition section](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties).
 
+You can also specify validation rules in the field `jsonSchema`, check the
```

Comment:
check -> check out?

Reply:
Can you provide an example here?

Reply:
I realized you added an example to `Parsing requests`.

Reply:
yeah I think it's better to have the example in both places.